ci-operator/step-registry/baremetalds/capi/gather/baremetalds-capi-gather-commands.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Why do we need to define capi-gather? can't we just use the assisted-gather?
The assisted-gather already collects the hypershift and capi provider logs for capi tests.

Reply:
There are different assisted-gathers for ZTP and test-infra. The one you mention is for test-infra and it's not relevant for this workflow here (even more, it's just not compatible)

Reply:
What do you mean by not compatible?
test-infra implement the gather logic we want, why can't we use it instead of implementing a gather script the capi repo?
Note that in order to debug we require the logs that the assisted-gather already collect (the service logs, assisted-realted CRs, hypershift logs and so on)

Reply:
We can, it was a tradeoff between refactoring gather logic in baremetalds-assisted and created our own... There is stuff like
```
cd /home/assisted
cp -v -R ./reports /tmp/artifacts || true 
```
and this structure of directories does not exist in the CAPI ZTP flow.

>we require the logs that the assisted-gather already collect (the service logs, assisted-realted CRs, hypershift logs and so on)

Service logs of Assisted do not live in the same place when you deploy in minikube and when you deploy via ACM... And so on. It is really not technical but purely logistics how much of a coupling we want to have between assisted-* and capi*

Reply:
Got it.
Let's reuse what we can, if it makes life easier or give us more flexibility we can define a capi specific gather